### ISSUE TITLE
Allow selection of virtual environment by providing the interpreter path

### ISSUE THREAD

**Issue by TrevorBenson** [microsoft/vscode-jupyter#15583]
When selecting a Jupyter Notebook Kernel from Python Environments provide an option to **Enter interpreter path...**.

----

### Reasoning

In VSCode selecting a virtual environment in the bottom/status bar for a regular Python file is available. This interface provides many Python Virtual Environments. However, it also allows providing the full path to `bin/python` of the virtual environment.

When selecting a Kernel for Jupyter notebooks from Python Environments the list of Virtual Env's provided may be much shorter than what VSCode displays for a regular python file. When this happens there may not be a way to select the existing Virtual Env used by the rest of the IDE for the project. It would be nice to have the same functionality for providing the entire path to the interpreter so creating a new Kernel/Environment is not the only remaining option.

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/15087

> When selecting a Kernel for Jupyter notebooks from Python Environments the list of Virtual Env's provided may be much shorter than what VSCode displays for a regular python file. When this happens there may not be a way to select the existing Virtual Env used by the rest of the IDE for the project. 

Please can you share a screenshot or provide more details on this.
I'm struggling to understand this second.

**Comment by TrevorBenson** [comment]
This does appear to be the same request, I searched initially on "environment", so missed the interpreter subject when looking for a duplicate. I will close this one with this comment and rely on #15087 

-----

Here are some screenshots.

What I see when selecting the interpreter from `file.py`:
![vscode-select-interpreter](https://github.com/microsoft/vscode-jupyter/assets/9167887/60900a6a-69a7-47f5-9f30-f7aa7aedb8f8)

*  I have around 2 dozen virtual environments and sometimes I see many more than the 5 shown currently.



What I see when selecting the Kernel from `file.ipynb`: 
![image](https://github.com/microsoft/vscode-jupyter/assets/9167887/281fc941-5c3b-4a34-9d32-4721fdbdc0b3)

* From what I recall when a `file.py` is selecting an interpreter I sometimes see 10-20 venvs However, I believe a `file.ipynb` selecting a kernel may show me fewer. At the moment only 5 were displayed for either so I cannot confirm the behavior difference I was describing.
  * Only a single venv is displayed in the select a kernel example because I used **Jupyter › Kernels: Exclude Python Environments** trying to see if this would update the list of virtual environments with a different 5. Instead, it removed 4 of the original 5, leaving only odr.
